examples/research_projects/intel_opts/inference_bf16.py

```diff
@@ -0,0 +1,49 @@
+import torch
```

Comment:
Could we add a README.md to `research_projects/intel_opts`?

Reply:
Done. Added some initial ones. Please review. Thanks.